opencensus/common/monitored_resource/k8s_utils.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from opencensus.common.monitored_resource import gcp_metadata_config
+
+_K8S_ATTRIBUTES = {
+    # ProjectID is the identifier of the GCP project associated with this
+    # resource, such as "my-project".
+    'project_id': 'project/project-id',
```

Comment:
`project-id` and `location` aren't part of the K8S resource. For K8S container resource we have:
- k8s.io/cluster/name
- k8s.io/namespace/name
- k8s.io/pod/name
- k8s.io/container/name

See the latest Java implementation: https://github.com/census-instrumentation/opencensus-java/blob/master/contrib/resource_util/src/main/java/io/opencensus/contrib/resource/util/K8sContainerResource.java.

I'll update the specs in a minute.

Reply:
Thanks, updated in baa06f1.